proarc-webapp/src/main/java/cz/cas/lib/proarc/webapp/client/presenter/DigitalObjectManager.java

```diff
@@ -303,6 +310,8 @@ private void initContextMenu(Menu menu, ActionSource actionSource) {
         menu.addItem(new MenuItemSeparator());
         menu.addItem(Actions.asMenuItem(deleteAction, actionSource, true));
         menu.addItem(Actions.asMenuItem(registerUrnNbnAction, actionSource, true));
+        menu.addItem(new MenuItemSeparator());
+        menu.addItem(Actions.asMenuItem(expandTreeAction, actionSource, false));
```

Comment:
Je to trochu jinak. Tečky se skutečně dodávají primárně do title. V případě, že je akce součástí menu. Jelikož by v toolbaru 3 tečky zabraly nepřiměřeně místo a někdy tam title vůbec není, tak se použijí v tooltipu. Viz. kontextové menu třeba ve Správě digitálních objektů.

FOXML je trochu zvláštní případ. Otevře novou stránku. Asi by tady být tečky nemusely, protože uživatel není dotazován. Jelikož už máme Code Mirror pro XML, tak by bylo pěkné FOXML zobrazit přímo v proarcu. To je ale na samostatné issue.